rules/standard_rules/malicious_sso_dns_lookup.yml

```diff
@@ -7,6 +7,7 @@ Enabled: false
 LogTypes:
   - CiscoUmbrella.DNS
   - Crowdstrike.DNSRequest
+  - Crowdstrike.FDREvent
```

Comment:
Since `FDREvent` events' count will be much higher now, are we sure this won't ... cause costs to go off-the-roof?

Reply:
Good callout. What does this rule run internally?

Reply:
[Searches p_any_domain names mostly](https://github.com/panther-labs/panther-analysis/blob/2f1e460f724f777fb6f3e3599a979e733e90cca5/rules/standard_rules/malicious_sso_dns_lookup.py)

Reply:
I don't think it's an issue, the only potential enhancement I see is converting the domain to lowercase only once in the loop instead of in every list comprehension or check:

```python
for ... :
  domain = domain.lower()
```

Reply:
this is not logically equal for the FDREvent type. `p_any_domain_names` is going to be filled out if any string in the `event` is conceivably a domain name. There is no filtering in this rule to limit to equivalent FDREvents that are the DNSQuery. 

I sampled one hour on an internal instance: 
* ~51,000 Crowdstrike.DNSRequest events
* ~997,000 Crowdstrike events across all types ( including DNSRequest )

Reply:
Agreed. Fixed on this one: https://github.com/panther-labs/panther-analysis/pull/648/commits/9b1119fba5e002051511b23cab31ff2fe63a9e63
Let me know what you think.

Reply:
looks good, thank you!